packages/front-end/components/Report/ConfigureReport.tsx

```diff
@@ -208,7 +208,10 @@ export default function ConfigureReport({
             <DimensionChooser
               value={form.watch("experimentAnalysisSettings.dimension") || ""}
               setValue={(v) =>
-                form.setValue("experimentAnalysisSettings.dimension", v)
+                form.setValue(
+                  "experimentAnalysisSettings.dimension",
+                  v ?? undefined
```

Comment:
On value we are using an empty string as fallback but here we are using undefined. Should we consolidate on one?

Reply:
Fixed it with a rewrite of how setValue works for this component.